src/tls13crypto.rs

```diff
@@ -237,15 +240,48 @@ pub fn to_libcrux_sig_alg(a: &SignatureScheme) -> Result<signature::Algorithm, T
 pub fn sign(
     alg: &SignatureScheme,
     sk: &Bytes,
+    cert: &Bytes,
     input: &Bytes,
     ent: Bytes,
```

Comment:
Please add a comment here that this needs to be reworked.

Reply:
I've made an issue for it, #72.